l3kernel/l3prg.dtx

```diff
@@ -175,7 +207,21 @@
 % \cs{prg_return_true:} \cs{else:} \cs{prg_return_false:} \cs{fi:} is
 % optimized.
 %
-% \begin{function}[updated = 2023-05-26]
+% \begin{function}[added = 2024-11-21]
+%   {
+%     \prg_new_eq_conditional:NN,
+%     \prg_set_eq_conditional:NN,
+%     \prg_gset_eq_conditional:NN
+%   }
+%   \begin{syntax}
+%     \cs{prg_new_eq_conditional:NNn} \cs[no-index]{\meta{name_1}:\meta{arg spec_1}} \cs[no-index]{\meta{name_2}:\meta{arg spec_2}}
```

Comment:
```suggestion
%     \cs{prg_new_eq_conditional:NN} \cs[no-index]{\meta{name_1}:\meta{arg spec_1}} \cs[no-index]{\meta{name_2}:\meta{arg spec_2}}
```

Reply:
Is it possible that new and old conditionals have different `arg spec`s? If not then need `s/arg spec_1/arg apec/g` and `s/arg spec_2/arg apec/g` replacements.

Reply:
I'll deal with that outside the PR - it's copy-pasted from the current text